api/v1beta1/foundationdbcluster_types.go

```diff
@@ -194,7 +194,7 @@ type FoundationDBClusterSpec struct {
 	// This must be a valid Kubernetes label value. See
 	// https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set
 	// for more details on that.
-	// +kubebuilder:validation:MaxLength=32
+	// +kubebuilder:validation:MaxLength=64
```

Comment:
TODO: Remove the link above. The prefix will be used a s value not as key.

Reply:
Another note we should not exceed 63 (this is the maximum for a label value). I would guess that `cluster-controller` (19 chars) is currently the longest role so we should be fine to have 63-20= 43 as `MaxLength`